app/models/like.rb

```diff
@@ -1,2 +1,8 @@
 class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+  after_save :update_likes_counter
+  def update_likes_counter
+    post.upadate(likes_counter: post.likes.count)
+  end
```

Comment:
- [ ] OPTIONAL
Following best practices `update_likes_counter` should be marked as private after implementing the `after_save` method.